warehouse/query-core/src/main/java/datawave/query/QueryParameters.java

```diff
@@ -144,6 +144,7 @@
     public static final String LIMIT_FIELDS = "limit.fields";
     
     public static final String GROUP_FIELDS = "group.fields";
+    public static final String GROUP_FIELDS_BATCH_SIZE = "group.fields.batch.size";
     public static final String UNIQUE_FIELDS = "unique.fields";
```

Comment:
controls the 'batch size' on the tservers. Purpose is to prevent OOM and to reduce wait time for the webserver. If value is zero, no grouping is done on the tservers and it is done only on the webserver